source/usage-examples/deleteOne.txt

```diff
@@ -0,0 +1,58 @@
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+You can delete a single document from a collection using the ``deleteOne()`` 
+method on a ``MongoCollection`` object. The method accepts a query filter
+that matches the document you want to delete. If you do not specify
+a filter, MongoDB matches the first document in the collection. The 
+``deleteOne()`` method only deletes the first document matched.
+
+This method returns an instance of ``DeleteResult`` which contains information 
+including how many documents were deleted as a result of the operation.
+
+If your delete operation fails, the driver raises an exception. For more
+information on the types of exceptions raised under specific conditions,
+see the API documentation for ``deleteOne()``, linked at the bottom of
+this page.
+
+Example
+-------
+
+The following snippet deletes a single document from the ``movies``
+collection of the ``sample_mflix`` database. The example uses the ``eq()`` 
+filter to match movies with the ``title`` exactly matching the text
+``'The Garbage Pail Kids Movie'``.
+
+.. include:: /includes/connect-guide-note.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/deleteOne.java
+   :language: java
+
+If you are using the **legacy API**, see the :doc:`FAQ </faq>` to determine 
+what changes you need to make to this code example.
```

Comment:
Should we put this in a note? Seems irrelevant for most users so we might wanna set it off visually.

Reply:
Sure. I was undecided whether this belongs in a note since I always thought notes were visual cues that drew the eye which would be placing more importance on that information rather than less.